$NetBSD$

* install perl modules into vendor
* escape tab for input of ed.
* enable clamav for the case --with-clamav=dir too

--- configure.orig	2016-07-01 03:31:03.000000000 +0000
+++ configure
@@ -21284,7 +21284,7 @@ $as_echo "#define HAVE_CLAMAV /**/" >>co
 fi
 
 
- if test "$with_clamav" = "yes"; then
+ if test "$with_clamav" != "no"; then
   HAVE_CLAMAV_TRUE=
   HAVE_CLAMAV_FALSE='#'
 else
@@ -27031,7 +27031,7 @@ fi
 
 
     PERL="${PERL}"
-    MAKE_MAKER_ARGS="PREFIX=${prefix}"
+    MAKE_MAKER_ARGS="INSTALLDIRS=vendor PREFIX=${prefix}"
 
 
 _ACEOF
@@ -28496,9 +28496,9 @@ _LT_EOF
 		n
 		s/\$(SITEARCHEXP)/\$(DESTINSTALLSITEARCH)/
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
 	  }' Makefile
     )
  ;;
@@ -28509,9 +28509,9 @@ _LT_EOF
 		n
 		s/\$(SITEARCHEXP)/\$(DESTINSTALLSITEARCH)/
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
 	  }' Makefile
     )
  ;;
@@ -28522,9 +28522,9 @@ _LT_EOF
 		n
 		s/\$(SITEARCHEXP)/\$(DESTINSTALLSITEARCH)/
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist"
 		a \
-	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
+\	$(RM_F) "$(DESTINSTALLSITEARCH)/perllocal.pod"
 	  }' Makefile
     )
  ;;
